core/test/integration/synchronization/ethereum_synchronization.cpp

```diff
@@ -268,3 +270,116 @@ TEST_F(EthereumLikeWalletSynchronization, XpubETCSynchronization) {
+            configuration->putString(api::Configuration::BLOCKCHAIN_EXPLORER_API_ENDPOINT, "http://test.test");
+            auto wallet = wait(pool->createWallet(walletName, "ethereum", configuration));
+            std::set<std::string> emittedOperations;
+            {
+                auto nextIndex = wait(wallet->getNextAccountIndex());
+                EXPECT_EQ(nextIndex, 0);
+
+                auto account = createEthereumLikeAccount(wallet, nextIndex, ETH_KEYS_INFO_LIVE);
+                auto waiter = createSyncReceiver();
+                test::UrlConnectionData blockNotFound;
+                blockNotFound.statusCode = 404;
+                blockNotFound.body = "Block 0xaef3a92f1017445c98139b7ab3ddd6e8abfe75588652338ca6f537cf47ab620d not found";
+                
+                fakeHttp->setBehavior({ 
+                    {
+                        "http://test.test/blockchain/v3/eth/blocks/current" ,
+                        test::FakeUrlConnection::fromString("{\"hash\":\"0xdd73566cf4913cba5060377397283cf411e03bdbe1b83d4695699d9e6ac1c941\",\"height\":8175344,\"time\":\"2019-07-18T14:54:00Z\",\"txs\":[]}")
+                    },
+                    {
+                        "http://test.test/blockchain/v3/eth/syncToken",
+                        test::FakeUrlConnection::fromString("{\"token\":\"f867afdf-c5b1-4a9d-b3e3-b4dfc985be8a\"}")
+                    },
+                    {
+                        "http://test.test/blockchain/v3/eth/addresses/0xabf06640f8ca8fC5e0Ed471b10BeFCDf65A33e43/transactions",
+                        test::FakeUrlConnection::fromString("\
+{\
+    \"truncated\": false,\
+    \"txs\": [\
+        {\
+            \"hash\": \"0x24ebd24caebcdb5d5fe22ba4d2e1904515dbb7d8fcd968789c8810ad379ba866\",\
+            \"status\": 1,\
+            \"received_at\": \"2018-06-08T10:19:10Z\",\
+            \"nonce\": \"0x05\",\
+            \"value\": 1000000000000000,\
+            \"gas\": 21000,\
+            \"gas_price\": 11000000000,\
+            \"from\": \"0xf0c91a8f0f95a2fd90ab74c6c13ef0db2af96447\",\
+            \"to\": \"0xabf06640f8ca8fC5e0Ed471b10BeFCDf65A33e43\",\
+            \"input\": \"0x\",\
+            \"cumulative_gas_used\": 6388806,\
+            \"gas_used\": 21000,\
+            \"transfer_events\": {\
+                \"list\": [],\
+                \"truncated\": false\
+            },\
+            \"actions\": [\
+                {\
+                    \"from\": \"0xf0c91a8f0f95a2fd90ab74c6c13ef0db2af96447\",\
+                    \"to\": \"0xabf06640f8ca8fC5e0Ed471b10BeFCDf65A33e43\",\
+                    \"value\": 1000000000000000\
+                }\
+            ],\
+            \"block\": {\
+                \"hash\": \"0x0160a3b2b762ec42f5a1134ac260a21c98f377e153d332efb6e20534abb37f78\",\
+                \"height\": 5752778,\
+                \"time\": \"2018-06-08T10:19:10Z\"\
+            }\
+        }\
+    ]\
+}")
+                    },
```

Comment:
I don’t get that indentation neither.

Reply:
Oh I actually get it. Can you indent that line up to the same level of the rest? It’s JSON, the parser won’t care.

Reply:
Change it.